src/components/ToDoList.js

```diff
@@ -0,0 +1,25 @@
+import ToDo from "./ToDo.js";
+
+export default function ToDoList({
+  toDos,
+  onToggleStatus,
+  onDeleteToDo,
+  handleNameChange,
```

Comment:
I'd be consistent with the naming:
`onToggleStatus`
`onDeleteTodo`
 and then you have `handleNameChange`

I'd rename that one to `onNameChanged` or something similar to be consistent with the rest of the props